services-custom/dynamodb-enhanced/it/main/java/software/amazon/awssdk/enhanced/dynamodb/EnhancedClientIntegrationTest.java

```diff
@@ -36,10 +34,13 @@
 import software.amazon.awssdk.services.dynamodb.model.ScalarAttributeType;
 import software.amazon.awssdk.services.dynamodb.model.TableStatus;
 import software.amazon.awssdk.testutils.Waiter;
+import software.amazon.awssdk.testutils.service.AwsTestBase;
 
-public class EnhancedClientIntegrationTest {
+public class EnhancedClientIntegrationTest extends AwsTestBase {
     private static final String TABLE = "books-" + UUID.randomUUID();
-    private static final DynamoDbClient dynamo = DynamoDbClient.create();
+    private static final DynamoDbClient dynamo = DynamoDbClient.builder()
+                                                               .credentialsProvider(CREDENTIALS_PROVIDER_CHAIN)
+                                                               .build();
```

Comment:
I think we need to pass dynamodb to line 78 and create an async dynamodb with credentialsProvider and pass it to line 97

Reply:
Oops. Thanks.